src/devices/LiquidLevel/LiquidLevelSwitch.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+
+namespace Iot.Device.LiquidLevel
+{
+    /// <summary>
+    /// Supports any single pin output digital liquid level switch which is configured
+    /// </summary>
+    public class LiquidLevelSwitch : IDisposable
+    {
+        private readonly int _dataPin;
+        private readonly PinValue _liquidPresentPinState;
+        private readonly bool _shouldDispose;
+
+        private GpioController _controller;
+
+        /// <summary>Creates a new instance of the LiquidLevelSwitch.</summary>
+        /// <param name="dataPin">The data pin</param>
+        /// <param name="liquidPresentPinState">Whether the pin value used to detect liquid is low or high</param>
+        /// <param name="pinNumberingScheme">Use the logical or physical pin layout</param>
+        /// <param name="gpioController">A Gpio Controller if you want to use a specific one</param>
+        /// <param name="shouldDispose">True to dispose the Gpio Controller</param>
+        public LiquidLevelSwitch(int dataPin, PinValue liquidPresentPinState, GpioController gpioController = null,
```

Comment:
generally for formatting args you either move all of them on new lines or none of them, i.e. here I'd do (aligned to current indent level):

```csharp
public LiquidLevelSwitch(
    int dataPin,
    PinValue liquidPresentPinState,
    GpioController gpioController = null,
    PinNumberingScheme pinNumberingScheme = PinNumberingScheme.Logical,
    bool shouldDispose = true)
{
...
```